lib/services/base.js

```diff
@@ -1,13 +1,18 @@
 const Stripe = require('stripe');
 
 module.exports = class BaseService {
-  constructor (options = {}) {
-    if (!options.secretKey) {
+  constructor (secretKey, config = {}) {
+    // If first parameter is an object, we assume the old behavior of passing in secretKey
+    if(typeof secretKey === 'object') {
+      secretKey = secretKey.secretKey;
+      this.paginate = secretKey.paginate;
+    }
+
+    if (!secretKey) {
       throw new Error('Stripe `secretKey` needs to be provided');
     }
 
-    this.stripe = Stripe(options.secretKey);
-    this.paginate = options.paginate = {};
+    this.stripe = Stripe(secretKey, config);
```

Comment:
before this line, you should handle if pagination is specified in the config, something like:
```
if (config.paginate) {
  this.paginate = config.paginate || {};
  delete config.paginate
}
```